### DISCUSSION TITLE
Unable to find DBT as an option when connecting to Services

### DISCUSSION THREAD

**Discussion by ShilpaVernekar** [open-metadata/OpenMetadata#5472] [Q&A] [answered]
Unable to find DBT as an option while setting up a connection to a database service.

[Originally answered in Slack support](https://openmetadata.slack.com/archives/C02B6955S4S/p1652883316107209)

**Comment by ShilpaVernekar** [comment, marked as answer]
The DBT UI was added to the ingestion pipeline in Release **0.10.1**. So, please ensure that you are using the latest version of OpenMetadata.

- Next, create a service connection to a database service. This will prepare the Entity with the proper connection information to your sources.
- Then, when preparing an ingestion via the UI, you will find a step as Configure DBT, with different options to pass the manifest and catalog files.

DBT is not a database service, so as a first step you'd need to prepare a service connection. For example, if you have DBT running on top of a MySQL instance. DBT gives some extra information from the tables already existing in a database service, rather than a specific source. You can read more about [DBT integration here](https://docs.open-metadata.org/connectors/ingestion/workflows/metadata/dbt).

**Reply by MichaelTiemannOSC** [reply]
DBT Integration page is now 404

**Comment by ShilpaVernekar** [comment]
Thanks for letting us know @MichaelTiemannOSC 
Here's the updated link to [DBT integration doc](https://docs.open-metadata.org/connectors/ingestion/workflows/metadata/dbt)

Updating the same here.

**Reply by divyapavar** [reply]
Hi Shilpa,

The updated link is also giving 404. Could you please provide the documentation link that is working. Thanks!

**Reply by ShilpaVernekar** [reply]
Hi @divyapavar, Thanks for letting me know. Please check this one - https://docs.open-metadata.org/connectors/ingestion/workflows/metadata/dbt
